runtime/compiler/codegen/J9CodeGenerator.hpp

```diff
@@ -390,7 +399,8 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGeneratorConnector
       {
       HasFixedFrameC_CallingConvention    = 0x00000001,
       SupportsMaxPrecisionMilliTime       = 0x00000002,
-      SupportsInlineStringCaseConversion  = 0x00000004 /*! codegen inlining of Java string case conversion */
+      SupportsInlineStringCaseConversion  = 0x00000004, /*! codegen inlining of Java string case conversion */
+      SupportsInlineStringIndexOf  = 0x00000008 /*! codegen inlining of Java string index of */
```

Comment:
The other enumerations seem to have "aligned" values. Can we fix the spacing here so as to not deviate?

Reply:
8ffd466